dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/internal/LibertyGradleConstants.java

```diff
@@ -27,4 +27,13 @@
 
     // preferences
     public static final String PROMPT_PREFERENCE = "libertyGradlePrompt";
+    
+    // tasks
+    public static final String BUILD_TASK = "build";
+    public static final String ASSEMBLE_TASK = "assemble";
+    public static final String INSTALL_APPS_TASK = "installApps";
+    
+    // arguments
+    public static final String[] SKIP_TESTS_ARGS = new String[] {"--exclude-task", "test"};
```

Comment:
I thought the option was -x ?

Reply:
--exclude-task is the same as -x, I used the long format since it is more explicit